boards/arm/nrf5340_audio_dk_nrf5340/doc/index.rst

```diff
@@ -0,0 +1,165 @@
+.. _thingy53_nrf5340:
+
+nRF5340 Audio DK
+################
+
+Overview
+********
+
+The nRF5340 Audio DK (PCA10121) is designed for showcasing, developing and experimenting
+with Bluetooth LE Audio. Keywords: Auracast™, broadcast, BIS, unicast, CIS, headsets, microphones.
```

Comment:
Seems redundant both write broadcast and BIS, as well as unicast and CIS. They are, in this context, the same, right?

Reply:
Addressed in my suggestion.

Reply:
That is true. Added this to match on keyword search.